.github/workflows/ci.yaml

```diff
@@ -88,7 +88,9 @@ jobs:
             if [[ ${{ matrix.os }} == "macos-latest" ]]; then
               export PYOPENCL_TEST=port:pthread
             fi
-            python -m pytest --durations=10 --tb=native --junitxml=pytest.xml --doctest-modules -rxsw .
+            python -m pytest --durations=10 --tb=native --junitxml=pytest.xml --doctest-modules -rxsw ./euler
+            python -m pytest --durations=10 --tb=native --junitxml=pytest.xml --doctest-modules -rxsw ./wave
+            python -m pytest --durations=10 --tb=native --junitxml=pytest.xml --doctest-modules -rxsw ./diffusion
```

Comment:
Won't moving the tests blow up the size of diffs in downstream PRs? (and/or cause merge conflicts?) I think we can just run each file separately for now, as a band-aid.

Reply:
This will make @majosm happy, too.  FWIW: I like the subdirectories better because it provides an easily discernible separator between the tests associated with each module and/or capability.

Reply:
1052c59